src/main/java/in/projecteka/consentmanager/user/PatientsController.java

```diff
@@ -61,8 +60,9 @@
                 .flatMap(caller -> transactionPinService.validatePinFor(caller.getUsername(), request.getPin(), request.getScope()));
     }
 
+    @ResponseStatus(HttpStatus.ACCEPTED)
     @PostMapping("/profile")
-    public Mono<Session> create(@RequestBody SignUpRequest request,
+    public Mono<Void> create(@RequestBody SignUpRequest request,
                                 @RequestHeader(name = "Authorization") String token) {
         var signUpRequests = SignUpRequestValidator.validate(request, userService.getUserIdSuffix());
         return signUpRequests.isValid()
```

Comment:
Can you keep this change as a separate commit which can be merged later? once app is done with their changes?

Reply:
We can return CREATED I believe.

Reply:
> Can you keep this change as a separate commit which can be merged later? once app is done with their changes?

Am not very sure about that since I added sessionId, implementation on top of the password optional commit. I will try ordering the local commit and see if there's not too much manual work

Reply:
The commits are not easily separable. Is it okay to have the password optional as a part of this PR itself?

Reply:
Let me try splitting the commits.